test/functional-test/test_channelprofile.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+import os
+import lib.in_out.reader as reader
+import lib.nickTracker as nickTracker
+import lib.util as util
+import lib.vis as vis
+from lib.analysis import network,channel
+from mock import patch
+
+
+class ChannelProfileTest(unittest.TestCase):
+
+    def setUp(self):
+        self.current_directory = os.path.dirname(os.path.realpath(__file__))
+        self.log_data_dir = self.current_directory+"/data/input/"
+        self.log_data_kubuntu = reader.linux_input(self.log_data_dir,["#kubuntu-devel"], '2013-01-01', '2013-01-31')
```

Comment:
An appropriate variable name would be `self.log_data_kubuntu_devel`. Please make the same change at other places in this module.

Reply:
done sir